ir_datasets/etc/downloads.json

```diff
@@ -3582,5 +3582,80 @@
     "size_hint": 248875419,
     "expected_md5": "e9c5b81c9df6fdc0e2986fa8ffb8ff12"
   }
+},
+
+"hc4": {
+  "train/topics": {
+    "url": "https://raw.githubusercontent.com/hltcoe/HC4/main/resources/hc4/train.topics.v1-0.jsonl",
+    "size_hint": 104898,
+    "expected_md5": "cf3a43c4085e28ce03f37704771e0e36"
+  },
+  "dev/topics": {
+    "url": "https://raw.githubusercontent.com/hltcoe/HC4/main/resources/hc4/dev.topics.v1-0.jsonl",
+    "size_hint": 81444,
+    "expected_md5": "4c34c546c3e90de4733c3c9411ce9c6d"
+  },
+  "test/topics": {
+    "url": "https://raw.githubusercontent.com/hltcoe/HC4/main/resources/hc4/test.topics.v1-0.jsonl",
+    "size_hint": 964434,
+    "expected_md5": "a311237913a7335d45fe261e9ff7f11e"
+  },
+  "fa/docs": {
+    "instructions": "[Placeholder]",
```

Comment:
Fill in placeholders here with basic instructions? (These are printed to the command line when the files are not present.)